src/extension/launchScenariosManager.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as path from "path";
+import * as fs from "fs";
+
+export interface IConfiguration {
+    name: string;
+    cwd: string | null | undefined;
```

Comment:
`string | null | undefined` => `cwd?: string`
The same for other lines

Reply:
Also maybe we don't even need `IConfiguration` and `ILaunchScenarios` interfaces? Thoughts?